packages/components/src/mobile/icon-button/README.md

```diff
@@ -0,0 +1,55 @@
+IconButton
+=============================
+
+Proved the IconButton style button for the mobile editor used in the inserter. 
```

Comment:
Is "Proved" a typo? I'm trying to understand this phrase 😅

Reply:
Yes that is a typo.